grails-app/views/inventoryItem/_showCurrentStock.gsp

```diff
@@ -89,7 +89,7 @@
                     <td colspan="6">
                         <div class="fade empty center">
                             <warehouse:message code="inventory.noItemsCurrentlyInStock.message"
-                                               args="[format.product(product:commandInstance?.product)]"/>
+                                               args="[format.product(product:commandInstance?.product).decodeHTML()]"/>
```

Comment:
`format.product()` will escape html characters, but warehouse.message takes an unescaped string as argument.

At this point I'm wondering if using taglib output as input to another taglib is an anti-pattern when special characters may be involved